### DISCUSSION TITLE
The presence doesn't update (Stuck on "In the main menu").

### DISCUSSION THREAD

**Discussion by ghost** [ThiaudioTT/hoi4-presence#34] [Q&A]
Related: #36

So I have Hearts of Iron IV for Steam, and I've followed all the steps, but when I run "runRPC.bat" the game launches, and the rich presence starts displaying itself on Discord, but it's stuck on "In the main menu" and doesn't display information about what country I am playing as, what year it is, etc once I actually get into a game. How do I fix this?

**Comment by ThiaudioTT** [comment]
> How do I fix this?

You need to pass at least one month in the game or save the game

**Reply by ghost** [reply]
I just tried to save the game and let 2 months pass, I still have the same issue... Is there anything else that could be the problem?

Also when I launch the game from Steam by pressing "Play", the RPC does not launch. The game launches normally without launching the RPC, so I launch the game by using the "runRPC" batch file inside of the game's directory. Is that where I am royally screwing myself over?

I also added "exePath": "runRPC.exe", to the "launcher-settings.json" file to see if that would make the RPC batch run when I launched the game from Steam, it did not.

Maybe I am just stupid, if that's the case I apologize.

![image](https://github.com/ThiaudioTT/hoi4-presence/assets/166869231/7b3a3bed-58a7-4353-90d3-eadaf36062ea)

**Reply by ghost** [reply]
![image](https://github.com/ThiaudioTT/hoi4-presence/assets/166869231/282d18e7-45f1-48e8-8cd6-9090c4ccbd41)
